### PR TITLE
Update license and copyright info

### PR DESCRIPTION
**Pull request by fasterit**

### CODE REVIEW

File: htop.c [hunk 1]
```diff
@@ -35,7 +35,8 @@ static void printVersionFlag(void) {
 
 static void printHelpFlag(void) {
    fputs("htop " VERSION "\n"
-         "Released under the GNU GPL.\n\n"
+	 COPYRIGHT "\n"
```
Comment (cgzones) [inline htop.c:38]: Is this a tab?

**Review by cgzones — COMMENTED** [review]
LGTM :+1:

File: htop.c (hunk 1 above)
Comment (fasterit) [reply htop.c:38]: Hehe, I fear yes. Old habbits...

Comment (fasterit) [reply htop.c]: Should be fixed, thanks for the nudge @cgzones

**Comment by BenBE** [comment]
Looking for files without mentioning GPLv2 like this `$ grep -L GPLv2 -R *.c *.h */*.c */*.h` gives:
```
CommandScreen.c
DateMeter.c
DateTimeMeter.c
EnvScreen.c
InfoScreen.c
XAlloc.c
CommandScreen.h
DateMeter.h
DateTimeMeter.h
EnvScreen.h
InfoScreen.h
Macros.h
XAlloc.h
darwin/Battery.c
solaris/Battery.c
unsupported/Battery.c
darwin/Battery.h
solaris/Battery.h
unsupported/Battery.h
```

**Pull request merged**